types/react-resizable/index.d.ts

```diff
@@ -44,6 +44,7 @@ export interface ResizableProps {
     onResizeStart?: (e: React.SyntheticEvent, data: ResizeCallbackData) => any;
     onResize?: (e: React.SyntheticEvent, data: ResizeCallbackData) => any;
     draggableOpts?: any;
+    resizeHandles?: any;
```

Comment:
Oops, missed this, sorry! It looks like the handles could actually be typed instead of just `any` according to https://github.com/STRML/react-resizable#props . After that LGTM!